src/Datadog.Trace.Tools.Runner/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "Datadog.Trace.Tools.Runner": {
       "commandName": "Project",
-      "commandLineArgs": "--crank-import=\"C:\\temp\\crank\\results.json\""
+      "commandLineArgs": "--dd-env=ci --env-vars=myVar1=myValue1,myVar2=myValue2 -- dotnet --info"
```

Comment:
```
--env-vars=myVar1=myValue1,myVar2=myValue2
```
This looks a little weird with the `=` serving two different functions. Doesn't `CommandLineParser` support using the same parameter multiple times and mapping that to an collection (instead of us splitting manually using commas)?
```
--env-vars myVar1=myValue1 --env-vars myVar2=myValue2
```

Reply:
I tried, but is not supported, also have some conflict with another IEnumerable<string> to collect the command to wrap + args.

Reply:
Maybe this works? `--dd-env ci --env-vars myVar1=myValue1,myVar2=myValue2 -- dotnet --info`?

Reply:
Yeah that works too